pkg/config/parser.go

```diff
@@ -285,14 +285,18 @@ func funcMap(tmpl *template.Template, giterminismManager giterminism_manager.Int
 	}
 
 	envFunc := funcMap["env"].(func(string) string)
-	funcMap["env"] = func(value interface{}) (string, error) {
+
+	funcMap["env"] = func(value interface{}, args ...string) (string, error) {
 		envName := fmt.Sprint(value)
 		if err := giterminismManager.Inspector().InspectConfigGoTemplateRenderingEnv(context.Background(), envName); err != nil {
 			return "", err
 		}
 
 		if !giterminismManager.LooseGiterminism() {
 			if _, exist := os.LookupEnv(envName); !exist {
+				for _, arg := range args {
+					return arg, nil
+				}
```

Comment:
done, but it looks little bit overhead
I don't need os.GetEnv at all in my case

Reply:
In your case it might be, but generally in Helm functions defaults applied on "Empty" condition, which includes empty string. Take "default" function or conditions in if-else statements for example.

Reply:
Давай по-русски если есть что по этой строке поправить еще)

Reply:
Плз, втыкни os.GetEnv, если default есть. Если лень, могу сам втыкнуть.

Reply:
Кинь suggestion плиз не догоняю)

Reply:
Переписал по своему усмотрению, @ilya-lesikov проверь

Reply:
Ок, чуть позже гляну, если чего не хватает поправлю. Спасибо.

Reply:
@ilya-lesikov  Есть планы на мерж?

Reply:
@ilya-lesikov Прости, совсем закопался тут с новым движком. Щас гляну